packages/kit/src/views/Market/MarketHomeV2/components/MarketTokenList/MarketWatchlistTokenList.tsx

```
@@ -30,6 +28,8 @@
   // Get watchlist from atom if not provided externally
   const [watchlistState] = useMarketWatchListV2Atom();
   const [selectedNetworkId] = useSelectedNetworkIdAtom();
+  const { recommendedTokens } = useMarketBasicConfig();
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)